src/js/controllers/disclaimer.js

```diff
@@ -32,6 +36,8 @@ angular.module('copayApp.controllers').controller('disclaimerController',
       });
     };
 
-    create();
-
+    if (!profileService.profile)
+      create();
+    else
+      applicationService.restart();
```

Comment:
This action will restart Copay while I just want to view the disclaimer from Settings->About->Disclaimer.